contents/blog/how-to-treat-your-pricing-like-a-product.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2023-04-09
+title: How to treat your pricing like a product (and why it matters)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - raquel-m-smith
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Product growth
+tags:
+  - Product engineers
+  - Guides
+
+---
+"Pricing is important" is common knowledge – price determines profit margin, which directly or indirectly determines nearly everything else. 
+
+What isn't common knowledge is twofold:
+
+1. Your pricing should be as much of a product as the thing you actually market and sell
+2. How to go about treating your pricing like a product
+
+Let’s start with number 1.
+
+## Your pricing should be a product
+
+I feel like this statement is easy to nod along with yet difficult to grok. What does it actually mean to treat your pricing like a product? What even is a product? Isn’t everything a product?
+
+Yes, kind of. But we’re talking specifically about software products, and I bet you know them when you see them – even if you’re not paying attention to the specifics. 
+
+Here’s a rough working version of how I might define a “product”:
```

Comment:
```suggestion
Yes, kind of. It's a "you know it when you see it" type thing with software products. 

Here’s a rough working version of how I might define a “product”:
```
I don't really get this sentence "But we’re talking specifically about software products, and I bet you know them when you see them – even if you’re not paying attention to the specifics." You already say "yes," it makes it a bit confusing to read. I like the "you know it when you see it" bit though, so rewriting to include that.

I like the